pylab/website/templates/website/event_details.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+{% load trans from i18n %}
+{% load markdown from websitetags %}
+
+
+{% block content %}
+
+<h1>{{ event.title }}</h1>
+
+<div class="pull-right">
+  <h3>{{ event.address }}</h3>
+  <iframe width="425" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"
+          src="{{ event.osm_map_link }}" style="border: 1px solid black"></iframe>
+</div>
+
+<h3>{{ event.starts|date:'Y-m-d' }}
+{% if event.starts.date == event.ends.date %}
+    {% if not event.hide_time %}
+      <span style='padding-left: 16px;'>{{ event.starts|date:'H:i' }} – {{ event.ends|date:'H:i' }}</span>
+    {% endif %}</h3>
```

Comment:
What would this show if `starts = 2015-07-29 23:00` and `ends = 2015-07-30 01:00`?